doc/decisions/error_message.md

```diff
@@ -0,0 +1,137 @@
+#Error message & handling concept
+
+## Problem
+
+The current error concept has disadvantages in following regards:
+
+- Too verbose error message
+- A lot of redundant errors
+- Hard to manage specification file
```

Comment:
Write more about the goals. What does "Hard to manage" mean? Do we want the specification to be more static or do we want that adding errors is without merge conflicts?